application/src/main/java/org/thingsboard/server/service/cf/ctx/state/CalculatedFieldCtx.java

```diff
@@ -516,7 +586,15 @@ public boolean hasContextOnlyChanges(CalculatedFieldCtx other) { // has changes
                 return true;
             }
         }
-        return scheduledUpdateIntervalMillis != other.scheduledUpdateIntervalMillis;
+        if (scheduledUpdateIntervalMillis != other.scheduledUpdateIntervalMillis) {
+            return true;
+        }
+        if (calculatedField.getConfiguration() instanceof RelatedEntitiesAggregationCalculatedFieldConfiguration thisConfig
+                && other.getCalculatedField().getConfiguration() instanceof RelatedEntitiesAggregationCalculatedFieldConfiguration otherConfig
+                && (thisConfig.getDeduplicationIntervalInSec() != otherConfig.getDeduplicationIntervalInSec() || !thisConfig.getMetrics().equals(otherConfig.getMetrics()))) {
+            return true;
+        }
+        return false;
```

Comment:
Why don't we handle update of `useLatestTs`? If `hasContextOnlyChanges` and `hasStateChanges` both return false - the states won't pick up the new config... Please cover this case with a test